tests/test_plugin.py

```diff
@@ -25,6 +31,16 @@ def workspace(tmpdir):
     return Workspace(uris.from_fs_path(str(tmpdir)), Mock())
 
 
+@pytest.fixture
+def config(workspace):
+    """Return a config object."""
+    cfg = Config(workspace.root_uri, {}, 0, {})
+    cfg._plugin_settings = {
+        "plugins": {"black": {"line_length": 88, "cache_config": True}}
```

Comment:
If configuration caching is enabled by default we should have a test that checks that it can be disabled.

Reply:
I added a test for this since it was part of your TODOs, see `test_cache_config` below.